python/ordinal-classifier.ipynb

```diff
@@ -0,0 +1,466 @@
+      ],
+      "text/plain": [
+       "MultiOutputClassifier(estimator=LogisticRegression(max_iter=1000), n_jobs=-1)"
+      ]
+     },
+     "execution_count": 15,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Train ordinal classification models for each rating variable\n",
+    "ordinal_clf = MultiOutputClassifier(LogisticRegression(max_iter=1000), n_jobs=-1)\n",
+    "ordinal_clf.fit(X_train_transformed, y_train)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 16,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Make predictions\n",
+    "y_pred = ordinal_clf.predict(X_test_transformed)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 17,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Evaluate model performance - MAE and RMSE\n",
```

Comment:
While LogisticRegression uses the word regression in the title, this is not a regression problem and MAE & RMSE are not the appropriate metrics to use unless there is some specific eccentricity for an Ordinal Classification task.